api/common.go

```diff
@@ -3,7 +3,7 @@ package api // import "github.com/docker/docker/api"
 // Common constants for daemon and client.
 const (
 	// DefaultVersion of the current REST API.
-	DefaultVersion = "1.47"
+	DefaultVersion = "1.48"
```

Comment:
I opened a quick PR to update the version, as there's a couple of PRs pending that need a new version (and I was working on some); https://github.com/moby/moby/pull/48476